h/migrations/versions/4ed50e7ac940_add_pubid_column_to_group_table.py

```diff
@@ -0,0 +1,61 @@
+"""Add pubid column to group table
+
+Revision ID: 4ed50e7ac940
+Revises: 5aab7692b5de
+Create Date: 2015-10-22 18:18:12.530467
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4ed50e7ac940'
+down_revision = '5aab7692b5de'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from h import pubid
+
+Session = sessionmaker()
+
+group = sa.table('group',
+    sa.column('id', sa.Integer),
+    sa.column('pubid', sa.Text),
+)
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = Session(bind=bind)
+
+    # Add the pubid column to the group table, but add it as nullable and
+    # without the unique constraint. This is an O(1) operation.
```

Comment:
Some combination of [the PostgreSQL docs](www.postgresql.org/docs/9.4/static/sql-altertable.html), a couple of great talks about PostgreSQL migrations ([1](russ.garrett.co.uk/talks/postgres-gds/), [2](https://vimeo.com/83756644)), and experience :stuck_out_tongue:.